### DISCUSSION TITLE
什麼也找不到🤷 #1.9.9.6

### DISCUSSION THREAD

**Discussion by Truu70** [xiaojieonly/Ehviewer_CN_SXJ#1700] [General]
![Screenshot_20241205_043324_EhViewer](https://github.com/user-attachments/assets/72bf25e9-79e4-416a-bb29-f28d34d2855a)

**Comment by YamiNoHikari** [comment]


**Reply by Truu70** [reply]
看來還是沒效

**Comment by jankinT** [comment]
是不是用的ex，我用ex就这样，试试e-h

**Comment by jankinT** [comment]
如果是ex进不去的话可以试试这样，先把内置host啥的全关了，再退出登录，然后使用VPN选一个欧美节点(我选的美国亲测有效)，再通过网页登录，重新登录之后看看cookie里最下面那一行是不是很长一串字符，如果是mystery就是失败了

**Reply by Er-Mox** [reply]
非常感谢！！终于解决问题了！

**Reply by Qssss1110** [reply]
请问网页怎么用cookie登录呀

**Reply by qingzhixing** [reply]
原来这样看失效啊！谢谢！

**Reply by hfhfts996** [reply]
感谢！

**Reply by jankinT** [reply]
更简单的方法，先复制cookie，退出登录，然后vpn换欧美节点，再重新通过cookie登录，从粘贴板导入cookie，把最下面那一行，无论是啥都给他删了，再登录，然后能进去了，之后过一下再试试换其他地方节点试试有没有用

**Reply by fand-alt** [reply]
我也相同的问题试了确实可以用了 但好像只能用节点看了，不能裸连了
